demo/main.js

```diff
@@ -949,12 +949,16 @@ shakaDemo.Main = class {
             advanced[drmSystem] = shakaDemo.Main.defaultAdvancedDrmConfig();
           }
           if ('videoRobustness' in params) {
-            advanced[drmSystem].videoRobustness = params['videoRobustness'];
+            advanced[drmSystem].videoRobustness =
+                params['videoRobustness'].split(',');
           }
           if ('audioRobustness' in params) {
-            advanced[drmSystem].audioRobustness = params['audioRobustness'];
+            advanced[drmSystem].audioRobustness =
+                params['audioRobustness'].split(',');
           }
         }
+
+        this.configure('drm.advanced', advanced);
```

Comment:
without this, it would read the advanced settings from the hash, but it would never actually set it back on the player.